lib/private/Files/Storage/Wrapper/Encryption.php

```diff
@@ -81,7 +56,7 @@ public function __construct(
 		?Util $util = null,
 		?LoggerInterface $logger = null,
 		?IFile $fileHelper = null,
-		$uid = null,
+		?string $uid = null,
```

Comment:
Good occasion to also migrate to property promotion?

```suggestion
		private ?string $uid = null,
```

Reply:
Good point, I missed that